app/models/search_builder.rb

```diff
@@ -12,5 +12,7 @@ class SearchBuilder < Blacklight::SearchBuilder
 
```

Comment:
It's not just you. This is super confusing, and I think the Sufia/Hyrax architecture itself is confused, due to different people working on it at different times with different ideas. I don't completely understand it, and think there may be no 'simple' explanation, the code is weird and different code does different things. 

I brought this up in a Hyrax issue, see  projecthydra-labs/hyrax#707
projecthydra-labs/hyrax#769